content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -103,30 +93,16 @@ Choose this option if your cluster doesn't have [Tiller](https://github.com/kube
 
 1. Render and apply Istio's core components:
 
-    {{< text bash >}}
+    * For a production environment use the recommended Helm values or customize as needed:
```

Comment:
It may be a little more clear if you explicitly mention this requires a larger cluster. Also very minor, but we may want to put this second - people may tend to use the first command offered

Reply:
You are right about the first option, but we still need to put production first.

The size of a cluster depends on the size of the app being installed and the number of replicas. We may need to document the min size required for Istio. Do we have data for this @mandarjog ?

Reply:
Please do not use bullets for branching paths in procedures. They complicate content structure and can cause confusion with readers. Instead modify the step to  inform readers that the command only works in production environments.
```suggestion
    1. Render and apply Istio's core components in a production environment using the recommended Helm values and customize as needed:
```

Reply:
@andraxylia I would also like us to doucment hte minimum requred for production defaults (memory and CPU count).  I don't know what these values are, but they have been increased recently and are likely to cause failures during deploy that are nearly silent.  This happend to me as i was testing this document, and it took me 20 minutes to figure out I didn't have enough worker cpus (I had 4 iirc).

Reply:
@rcaballeromx Your suggestion does not work for hierarchical bullets, it displays an actual '1.'

@sdake Please submit a separate PR for further improvements and let's get this in.